app/services/browser_support.rb

```diff
@@ -0,0 +1,62 @@
+class BrowserSupport
+  @cache = LruRedux::Cache.new(1_000)
+
+  BROWSERSLIST_TO_BROWSER_MAP = {
+    and_chr: :chrome?,
+    and_uc: :uc_browser?,
+    chrome: :chrome?,
+    edge: :edge?,
+    firefox: :firefox?,
+    ios_saf: :safari?,
```

Comment:
ditto for `browser.ios? && browser.safari?`

Reply:
> ditto for `browser.ios? && browser.safari?`

Added as part of 86c4df0c1.